api/src/user/user.controller.ts

```diff
@@ -0,0 +1,126 @@
+import {
+  Body,
+  Controller,
+  Delete,
+  Get,
+  HttpException,
+  Param,
+  ParseIntPipe,
+  Post,
+  Put,
+  Query,
+} from '@nestjs/common';
+import { Prisma } from '@prisma/client';
+import { UserService } from './user.service';
+
+@Controller('user')
```

Comment:
Usually we use the plural to represent a resource.

Reply:
I note this, for the moment I leave it as it is, not to impact too much the work of Alexandre